app/middlewares/user.js

```diff
@@ -34,3 +36,13 @@ exports.checkOrder = [
     .customSanitizer(value => value && value.toUpperCase())
     .isIn([ascOrder, descOrder])
 ];
+
+exports.checkBoughtAlbumsPermission = (req, res, next) => {
+  const isAdmin = req.userPayload.role === admin_role;
+  const isRegularAndIsAllowedRequest =
```

Comment:
Could be more expressive like `isOwner`, check regular user in this point is meaningless because role is checked in first condition